### PR TITLE
RobloxCorporation.RobloxBootstrapper version Latest

### PR DESCRIPTION
**Pull request by Twig6943**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/101346)

### CODE REVIEW

File: manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: RobloxCorporation.RobloxBootstrapper
+PackageVersion: Latest
```
Comment (Trenly) [inline manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.installer.yaml:5]: Latest is not a valid package version. Please use a semantic version, even if it is arbitrary

File: manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: RobloxCorporation.RobloxBootstrapper
```
Comment (Trenly) [inline manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.installer.yaml:4]:
```suggestion
PackageIdentifier: RobloxCorporation.Roblox
```
No need to call out the fact it's a boostrapper. The folder and file names will need to be updated with this change.

File: manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: RobloxCorporation.RobloxBootstrapper
```
Comment (Trenly) [inline manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: RobloxCorporation.Roblox
```
No need to call out the fact it's a boostrapper. The folder and file names will need to be updated with this change.

File: manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: RobloxCorporation.RobloxBootstrapper
+PackageVersion: Latest
+PackageLocale: en-US
+Publisher: Roblox Corporation
+PackageName: Roblox
+License: Commercial
+Copyright: Copyright © 2020 Roblox Corporation. All rights reserved.
+ShortDescription: Roblox is an online gaming platform and game creation system developed by Roblox Corporation. It allows users to program games and play games created by other users.
```
Comment (Trenly) [inline manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.locale.en-US.yaml:11]:
```suggestion
ShortDescription: Roblox is an online gaming platform and game creation system developed by Roblox Corporation. It allows users to program games and play games created by other users.
Moniker: roblox
```

File: manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: RobloxCorporation.RobloxBootstrapper
```
Comment (Trenly) [inline manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.yaml:4]:
```suggestion
PackageIdentifier: RobloxCorporation.Roblox
```
No need to call out the fact it's a boostrapper. The folder and file names will need to be updated with this change.

File: manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: RobloxCorporation.RobloxBootstrapper
+PackageVersion: Latest
```
Comment (Trenly) [inline manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.locale.en-US.yaml:5]: Latest is not a valid package version. Please use a semantic version, even if it is arbitrary

File: manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.yaml [hunk 7]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: RobloxCorporation.RobloxBootstrapper
+PackageVersion: Latest
```
Comment (Trenly) [inline manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.yaml:5]: Latest is not a valid package version. Please use a semantic version, even if it is arbitrary

File: manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.locale.en-US.yaml [hunk 8]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: RobloxCorporation.RobloxBootstrapper
+PackageVersion: Latest
+PackageLocale: en-US
+Publisher: Roblox Corporation
+PackageName: Roblox
```
Comment (Trenly) [inline manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.locale.en-US.yaml:8]:
```suggestion
PackageName: Roblox Player
```
This should match the values written to control panel

**Review by Trenly — CHANGES_REQUESTED** [review]
As Stephen mentioned, an unattended install is required before this can be added

**Comment by Twig6943** [comment]
msftbot/changesRequested

**Comment by Twig6943** [comment]
hey why isnt this approved?

File: manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.yaml (hunk 7 above)
Comment (Twig6943) [reply manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.yaml:5]: apply change

File: manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.locale.en-US.yaml (hunk 8 above)
Comment (Twig6943) [reply manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.locale.en-US.yaml:8]: apply change

File: manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.installer.yaml (hunk 1 above)
Comment (Twig6943) [reply manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.installer.yaml:5]: apply change

File: manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.installer.yaml (hunk 2 above)
Comment (Twig6943) [reply manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.installer.yaml:4]: apply change

File: manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.locale.en-US.yaml (hunk 3 above)
Comment (Twig6943) [reply manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.locale.en-US.yaml:4]: apply change

File: manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.locale.en-US.yaml (hunk 4 above)
Comment (Twig6943) [reply manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.locale.en-US.yaml:11]: apply change

File: manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.yaml (hunk 5 above)
Comment (Twig6943) [reply manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.yaml:4]: apply change

File: manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.locale.en-US.yaml (hunk 6 above)
Comment (Twig6943) [reply manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.locale.en-US.yaml:5]: apply change

**Pull request closed without merging**